clang/lib/DPCT/APINamesGraphicsInterop.inc

```diff
@@ -0,0 +1,96 @@
+                                "experimental::map_resources",
+                            ARG(0), ARG(1), ARG(2))),
+        CALL_FACTORY_ENTRY("cudaGraphicsMapResources",
+                       CALL(MapNames::getDpctNamespace() +
+                                "experimental::map_resources",
+                            ARG(0), ARG(1)))),
+    UNSUPPORT_FACTORY_ENTRY("cudaGraphicsMapResources", Diagnostics::TRY_EXPERIMENTAL_FEATURE,
+                            ARG("cudaGraphicsMapResources"),
+                            ARG("--use-experimental-features=bindless_images"))))
+
+ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
+    UseExtBindlessImages,
+    ASSIGNABLE_FACTORY(ASSIGN_FACTORY_ENTRY(
+        "cudaGraphicsResourceGetMappedMipmappedArray", DEREF(0),
+        MEMBER_CALL(ARG(1), true, "get_mapped_mipmapped_array"))),
+    UNSUPPORT_FACTORY_ENTRY("cudaGraphicsResourceGetMappedMipmappedArray", Diagnostics::TRY_EXPERIMENTAL_FEATURE,
+                            ARG("cudaGraphicsResourceGetMappedMipmappedArray"),
+                            ARG("--use-experimental-features=bindless_images"))))
+
+ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
+    UseExtBindlessImages,
+    ASSIGNABLE_FACTORY(ASSIGN_FACTORY_ENTRY(
+        "cudaGraphicsSubResourceGetMappedArray", DEREF(0),
+        MEMBER_CALL(ARG(1), true, "get_sub_resource_mapped_array", ARG(2), ARG(3)))),
+    UNSUPPORT_FACTORY_ENTRY("cudaGraphicsSubResourceGetMappedArray", Diagnostics::TRY_EXPERIMENTAL_FEATURE,
+                            ARG("cudaGraphicsSubResourceGetMappedArray"),
+                            ARG("--use-experimental-features=bindless_images"))))
+
+ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
+    UseExtBindlessImages,
+    CONDITIONAL_FACTORY_ENTRY(
+        makeCheckNot(CheckArgIsDefaultCudaStream(2)),
+        CALL_FACTORY_ENTRY("cudaGraphicsUnmapResources",
+                       CALL(MapNames::getDpctNamespace() +
+                                "experimental::unmap_resources",
+                            ARG(0), ARG(1), ARG(2))),
+        CALL_FACTORY_ENTRY("cudaGraphicsUnmapResources",
+                       CALL(MapNames::getDpctNamespace() +
+                                "experimental::unmap_resources",
+                            ARG(0), ARG(1)))),
+    UNSUPPORT_FACTORY_ENTRY("cudaGraphicsUnmapResources", Diagnostics::TRY_EXPERIMENTAL_FEATURE,
+                            ARG("cudaGraphicsUnmapResources"),
+                            ARG("--use-experimental-features=bindless_images"))))
+
+ASSIGNABLE_FACTORY(CONDITIONAL_FACTORY_ENTRY(
+    UseExtBindlessImages,
+    DELETER_FACTORY_ENTRY("cudaGraphicsUnregisterResource", ARG(0)),
+    UNSUPPORT_FACTORY_ENTRY("cudaGraphicsUnregisterResource", Diagnostics::TRY_EXPERIMENTAL_FEATURE,
+                            ARG("cudaGraphicsUnregisterResource"),
+                            ARG("--use-experimental-features=bindless_images"))))
+
+// clang-format on
```

Comment:
The file without clang-format seems like not easier to read compared with non-format. Please try format and not format the file, and compare which one is better.